modules/crm_core_match/src/Matcher/MatcherListBuilder.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\crm_core_match\Matcher\MatcherListBuilder.
+ */
+
+namespace Drupal\crm_core_match\Matcher;
+
+use Drupal\Core\Config\Entity\ConfigEntityListBuilder;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Defines a class to build a listing of matcher config entities.
+ *
+ * @see \Drupal\crm_core_match\Entity\Matcher
+ */
+class MatcherListBuilder extends ConfigEntityListBuilder {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildHeader() {
+    // Overrides the original Header completely.
+    $header['label'] = $this->t('Label');
+    $header['description'] = $this->t('Description');
+    $header['plugin'] = $this->t('Matching engine (plugin)');
+
+    return $header + parent::buildHeader();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildRow(EntityInterface $entity) {
+    /** @var \Drupal\crm_core_match\Entity\Matcher $entity */
+    $row['label'] = $entity->getLabel();
+    $row['description'] = $entity->getDescription();
```

Comment:
Hm. See above about description. What might be more useful than this might be a $this->getPlugin()->getSummary() method, similar to field formatter configuration example. That could list to which contact types it applies, which fields are enabled for matching, something like that..